docs/examples/datasets/pipelines.py

```diff
@@ -350,3 +350,91 @@ def test_multiple_pipelines(client):
     )
     assert df.shape == (4, 4)
     assert found.sum() == 4
+
+
+# docsnip aggregate_with_str_window
+@meta(owner="data-eng-oncall@fennel.ai")
+@dataset
+class ActivityGroupedDataset:
+    timestamp: datetime
+    user_id: int = field(key=True)
+    amount_forever: float
+    amount_1d: float
+
+    @pipeline(version=1)
+    @inputs(Activity)
+    def create_grouped_dataset(cls, activity: Dataset):
+        return (
+            activity.dropnull(columns=["amount"])
```

Comment:
IIRC, `dropnull` can be called without using kwargs e.g. `dropnull('amount')`. Could you please check if that's true? If so, could you please use that syntax here?

In general, we obsess about keeping as few keystrokes/words in the docs/examples as possible because that leads to a better developer experience eventually (after all people just copy/paste docs examples).

Reply:
makes sense